src/components/Auth/ChangePassword/ChangePassword.css

```diff
@@ -29,3 +28,13 @@
 .forgot:hover {
   text-decoration: underline;
 }
+
+.submitbtn{
+  float: left;
+  max-width: 300px;
+  margin: 0 auto;
+}
+
+.label {
```

Comment:
This additional property is not needed when you already have a style object for labels